src/pages/patientView/timeline2/TimelineWrapperStore.tsx

```diff
@@ -0,0 +1,81 @@
+import { action, observable } from 'mobx';
+import _ from 'lodash';
+import * as React from 'react';
+import { Mutation } from 'cbioportal-ts-api-client';
+import { MutationStatus } from '../mutation/PatientViewMutationsTabUtils';
+
+export default class TimelineWrapperStore {
+    @observable groupByOption: Readonly<string> | null = null;
+
+    @observable vafChartHeight: Readonly<number> = 240;
+
+    @observable onlyShowSelectedInVAFChart:
+        | Readonly<boolean>
+        | undefined = undefined;
+
+    @observable vafChartLogScale: Readonly<boolean> | undefined = undefined;
+
+    @observable vafChartYAxisToDataRange:
+        | Readonly<boolean>
+        | undefined = undefined;
+
+    @observable tooltipModel: {
```

Comment:
unless you are really intending to update individual properties of this model (leaving others untouched) this should be @observable.ref   (see comment below)

Reply:
Fixed